stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/TextFieldUI.kt

```diff
@@ -455,6 +463,52 @@ private fun TrailingDropdown(
     }
 }
 
+private fun Modifier.onPreviewKeyEvent(
+    value: String,
+    focusManager: FocusManager,
+    direction: FocusDirection,
+): Modifier = onPreviewKeyEvent { event ->
+    if (event.type == KeyEventType.KeyDown &&
+        event.nativeKeyEvent.keyCode == KeyEvent.KEYCODE_DEL &&
+        value.isEmpty()
+    ) {
+        focusManager.moveFocus(direction)
+        true
+    } else {
+        false
+    }
+}
+
+/*
+ * Using 'composed' is no longer recommended
+ * https://developer.android.com/jetpack/compose/custom-modifiers#create_a_custom_modifier_using_a_composable_modifier_factory
+ */
+@SuppressLint("ComposableModifierFactory")
```

Comment:
Are we missing anything in our codebase? The official docs say that the lint rule has been removed, but it seems to still be around.

Reply:
I think it was removed in `Compose 1.6`. The `autofill` composable modifier in our current Compose version also suppresses this lint rule.